lte/gateway/c/oai/tasks/mme_app/mme_app_defs.h

```diff
@@ -167,8 +167,8 @@ int mme_app_handle_ue_context_modification_timer_expiry(
 void mme_app_handle_enb_reset_req(
     const itti_s1ap_enb_initiated_reset_req_t* const enb_reset_req);
 
-int mme_app_handle_initial_paging_request(
-    mme_app_desc_t* mme_app_desc_p, const char* imsi);
+imsi64_t mme_app_handle_initial_paging_request(
+    mme_app_desc_t* mme_app_desc_p, itti_s11_paging_request_t paging_req);
```

Comment:
change the signature as `const itti_s11_paging_request_t const* paging_req` and modify the calls to this function accordingly.